src/server/cluster/cluster_config_test.cc

```diff
@@ -57,34 +57,39 @@ TEST_F(ClusterConfigTest, ConfigSetInvalidMissingSlots) {
   EXPECT_EQ(ClusterConfig::CreateFromConfig(
                 kMyId, {{.slot_ranges = {{.start = 0, .end = 16000}},
                          .master = {.id = "other", .ip = "192.168.0.100", .port = 7000},
-                         .replicas = {}}}),
+                         .replicas = {},
+                         .migrations = {}}}),
```

Comment:
all the tests here are for empty migrations array
please add test for valid and invalid configurations